modules/server/src/main/java/org/jboss/wsf/stack/cxf/interceptor/HandlerAuthInterceptor.java

```diff
@@ -55,13 +56,28 @@
 public class HandlerAuthInterceptor extends AbstractPhaseInterceptor<Message>
 {
    private static final String KEY = HandlerAuthInterceptor.class.getName() + ".SECURITY_EXCEPTION";
-   
+   private boolean skipAuth = false;
```

Comment:
Consider making this `final` if possible:
`private boolean skipAuth;`

and then you set that field in both constructors.